packages/react-magma-dom/src/components/DatePicker/index.tsx

```diff
@@ -347,12 +426,34 @@ export const DatePicker = React.forwardRef<HTMLInputElement, DatePickerProps>(
 
     function handleInputBlur(event: React.FocusEvent) {
       const { value } = inputRef.current;
-      const day = parse(value, i18n.dateFormat, new Date());
+      const day = parse(
+        additionalContent ? value.split(' ')[0] : value,
+        i18n.dateFormat,
+        new Date()
+      );
 
-      if (isValidDateFromString(value, day)) {
-        handleDateChange(day, event);
+      let isValidDay;
+
+      if (additionalContent) {
```

Comment:
instead of using additionalContent, should we be more specific like `isDateTimePicker`?? Up to you, just thinking that I don't think there's any other possible usage of this additionalContent space within a datepicker so we could make this extra readable for ourselves